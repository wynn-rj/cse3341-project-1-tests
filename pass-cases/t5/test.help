Make sure the chosen case expression is assigned to the case variable (z)
