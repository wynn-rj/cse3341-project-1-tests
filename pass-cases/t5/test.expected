program
  int x,y,z;
begin
  input x,y;
  z:=x+y;
  case z of
    1:x+7
    |3,5:y*(x-z-27)
    |7:y-2
    else -266471638
  end;
  output z;
end
92
